components/EditMembers.js

```diff
@@ -51,6 +51,14 @@ class EditMembers extends React.Component {
       },
       'user.email.label': { id: 'user.email.label', defaultMessage: 'email' },
       'user.since.label': { id: 'user.since.label', defaultMessage: 'since' },
+      'members.remove.confirm.messageBegin': {
```

Comment:
This should be only one message including the variables in it.

Like `Do you really want to remove {userName} ({email}) from the core contributors of the collective?`

See https://docs.opencollective.com/help/developers/translations#dont-assume-words-order-stays-the-same-in-other-languages

Reply:
Alright.. Thanks for the feedback.. working on it right now

Reply:
I just made this fix

Reply:
Have you tested that it works as expected? Seems there is a typo.

Reply:
Oh! 😅 Found it! Fixing it.. You guys are awesome @znarf @Betree